examples/sw/log.27Oct21.mos2.icc.1.lmp

```diff
@@ -0,0 +1,109 @@
+LAMMPS (27 Oct 2021)
```

Comment:
Please see the file name conventions for log files as they are used in other examples.

Reply:
Renamed the files.